MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -437,5 +440,15 @@ private List<string> SplitWithCustomDelimiter(string val, char[] tagDelimiters,
 
             return items;
         }
+
+        private static string[] SplitWithWorkaround(string val, char separator, bool useWorkaround)
+        {
+            if (useWorkaround)
+            {
+                val = val.Replace(InternalValueSeparator.ToString(), AtlId3V22Separator.ToString(), StringComparison.OrdinalIgnoreCase);
```

Comment:
Why `ToString`?
Shouldn't we use exact match?
```suggestion
                val = val.Replace(InternalValueSeparator, AtlId3V22Separator);
```

Reply:
because val is string and replace does not take char as input

Reply:
We don't need to set string comparison since we know what both characters are and neither have case issues, so we can do the char replacement as suggested.